ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -60,6 +62,19 @@ import           Ouroboros.Network.Subscription.Dns
 import           Ouroboros.Network.Subscription.Worker (LocalAddresses (..))
 import           Ouroboros.Network.Tracers
 
+data DiffusionInitializationTracer
+  = DiffusionInitiatilizationRunServer
```

Comment:
I'd be in favour of shorter names, without `DiffusionInitialisation` prefix.  We can require that this module is imported qualified (that's my plan anyway).  The prefix will be in the logs because we log the tracer name, so no need to have it twice.